airflow/providers/amazon/aws/secrets/ssm.py

```diff
@@ -21,9 +21,10 @@
 from typing import List, Optional
 
 import boto3
+from cached_property import cached_property
 
 from airflow.models import Connection
-from airflow.secrets import CONN_ENV_PREFIX, BaseSecretsBackend
```

Comment:
`CONN_ENV_PREFIX` should probably be moved to `airflow/secrets/environment_variables.py` since it is no longer a shared pattern, but something specific to the env vars backend...

Reply:
Fixed